src/components/SearchBar.js

```diff
@@ -9,7 +9,7 @@ class SearchBar extends Component {
     formSubmit = (e) => {
         //execute some function to update locationQuery
         e.preventDefault();
-        this.props.locQryUpdt(e.target.city.value.toLowerCase());
+        this.props.locQryUpdt(e.target.location.value.toLowerCase());
```

Comment:
I think this will not need toLower because of the API's built-in search feature.